cli/electron.ts

```diff
@@ -24,14 +24,14 @@ function errorOut(msg: string): Promise<void> {
 
 export function electronAsync(parsed: p.ParsedCommand): Promise<void> {
     // Ensure there is a subcommand
-    let subcommand = parsed.arguments[0];
+    const subcommand = parsed.arguments[0];
 
     if (!subcommand) {
         errorOut("Please specify a subcommand");
     }
 
     // Validate current target
-    let needsCurrentTarget = (subcommand !== "build" && subcommand !== "dist") || !parsed.flags["release"];
+    const needsCurrentTarget = (subcommand !== "build" && subcommand !== "dist") || !parsed.flags["release"];
```

Comment:
Oops these 2 commands don't exist anymore; there is only "package", with the option "--installer". I need to update this check